sklearn/feature_selection/univariate_selection.py

```diff
@@ -134,6 +134,11 @@ def f_classif(X, y):
 
     pval : array, shape = [n_features,]
         The set of p-values.
+
+    See also
+    --------
+    chi2: Chi-squared statistic between label/feature for classification task.
```

Comment:
Sorry for missing it. 

In addition to "See also" references, I also updated chi2's summary of the function docstrings (not clear before).